clients/admin-ui/src/pages/systems/configure/[id].tsx

```diff
@@ -38,6 +43,30 @@ const ConfigureSystem: NextPage = () => {
     dispatch(setActiveSystem(system));
   }, [system, dispatch]);
 
+  useEffect(() => {
+    const status = router.query.status;
```

Comment:
After authorizing a connection, the user is redirected to the system page. The status query param allows us to show the appropriate toast message to the user.